gapic/src/main/com/google/gapid/widgets/Theme.java

```diff
@@ -160,6 +162,15 @@
 
   @RGB(argb = 0xffcccccc) public Color statusBarMemoryBar();
 
+  @RGB(argb = 0xff000000) public Color tabTitle(); // TODO: should be system defined
```

Comment:
Is this TODO comment left on-purpose?

Reply:
Also, I confirm it builds and is usable (and looks good!) on Linux, it would be nice to double check it works fine on Win and Mac.

Reply:
Yes, maybe we can figure out how to do it in the future, or discard it. For now, this is fine.